<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">bacground-color: #0E1920</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #0E1920</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comL">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>6</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;

</string>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openB">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>6</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #ff8bf0;
}</string>
       </property>
       <property name="text">
        <string>open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeB">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>6</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #ff8bf0;
}</string>
       </property>
       <property name="text">
        <string>close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="onOffBox">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>112</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="c1B">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>Цвет 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c2B">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>Цвет 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c3B">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>Цвет 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c4B">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>Цвет 4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c5B">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>Радуга</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c6B">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>Радуга + аниме</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c7B">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>Смена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c8B">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>Мигалки</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>148</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="c1B_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>Столбик З/К</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c2B_2">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>Столбик Радуга</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c3B_2">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>5 Полос</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c4B_2">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>3 Полосы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c5B_2">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>1 Полоса</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c6B_2">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>Старбоскоп</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c7B_2">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>Бегущие частоты</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c8B_2">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>Анализатор спектра</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="c1B_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c1B_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>25</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c1B_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>50</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c1B_7">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>75</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c1B_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color: #0E1920;
 color: white;
 border: 1px;
 border-radius:5;
}

QPushButton:pressed {
 background-color: #252b55;
}</string>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
